Pipfile

```diff
@@ -30,7 +30,7 @@ xlsxwriter = "==1.0.2"
 retrying = "==1.3.3"
 sdc-rabbit = "==1.5.1"
 alembic = "==0.9.6"
-pycryptodome = "==3.4.6"
+pycryptodome = "*"
```

Comment:
is there a reason for not tagging it to a specific version?

Reply:
It stopped working on the old version :L I'll add the specific newer version to the Pipfile.
There's probably a wider conversation to have about what dependencies we're locking in this file and which one's we'll let the Pipfile.lock take care of